src/app/theme.cpp

```diff
@@ -48,6 +52,8 @@ Theme::Theme() : QObject(qApp), palette()
     QFontDatabase::addApplicationFont(":/fonts/Montserrat/Montserrat-LightItalic.ttf");
     QFontDatabase::addApplicationFont(":/fonts/Montserrat/Montserrat-Regular.ttf");
 
+    qApp->setFont(Theme::font_14);
```

Comment:
this should be removed (moved before i merged pages into develop, probs came back when you were resolving conflicts hah)